docs/rest-apis/devportal/schemas.md

```
@@ -782,20 +807,36 @@ continued
+    "client_name": "my-app",
+    "grant_types": [
+      "client_credentials"
+    ]
+  }
 }
 
 ```
 
+OAuth key mapping entry attached to an application.
+
 ### Properties
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)